src/app/build.gradle

```diff
@@ -22,55 +22,46 @@ apply plugin: 'application'
 mainClassName = "org.vaccineimpact.orderlyweb.app_start.SparkAppKt"
 
 dependencies {
-    compile group: 'commons-io', name: 'commons-io', version: '2.6'
-    compile "org.jetbrains.kotlin:kotlin-reflect:$kotlin_version"
-
-    compile "com.sparkjava:spark-core:2.7.1"
-    compile "com.sparkjava:spark-template-freemarker:2.7.1"
-    compile "org.slf4j:slf4j-simple:1.7.25"
-
-    compile "com.google.code.gson:gson:2.2.4"
-    compile "com.github.salomonbrys.kotson:kotson:2.5.0"
-    compile "com.opencsv:opencsv:3.9"
-
-    compile "org.apache.commons:commons-lang3:3.12.0"
-    compile "org.apache.commons:commons-text:1.9"
-
-    compile "org.postgresql:postgresql:9.4.1212.jre7"
-    compile "org.jooq:jooq:3.9.1"
-    compile "org.jooq:jooq-meta:3.9.1"
-
-    compile "org.pac4j:spark-pac4j:3.0.0"
-    compile "org.pac4j:pac4j-http:3.2.0"
-    compile 'org.pac4j:pac4j-jwt:3.2.0'
-    compile 'org.pac4j:pac4j-oauth:3.6.1'
-    compile 'commons-codec:commons-codec:1.10'
-
-    compile "org.json:json:20201115"
-    compile "com.squareup.okhttp3:okhttp:4.8.1"
-
-    compile "org.ocpsoft.prettytime:prettytime:4.0.5.Final"
-
-    compile "org.kohsuke:github-api:1.303"
-
-    testCompile "org.slf4j:slf4j-simple:1.7.21"
-    testCompile "com.beust:klaxon:0.31"
-    testCompile "com.opencsv:opencsv:3.9"
-
-    testCompile "com.github.fge:json-schema-validator:2.2.6"
-
-    compile project(":databaseInterface")
-
-    testCompile "junit:junit:4.12"
-    testCompile "org.mockito:mockito-core:2.+"
-    testCompile "com.nhaarman:mockito-kotlin:1.3.0"
-    testCompile "org.assertj:assertj-core:3.6.2"
-    testCompile "com.beust:klaxon:0.31"
-    testCompile "net.sourceforge.htmlunit:htmlunit:2.4"
-    testCompile "org.xmlmatchers:xml-matchers:0.10"
-    testCompile "org.jsoup:jsoup:1.11.3"
+    implementation group: 'commons-io', name: 'commons-io', version: '2.6'
+    implementation "org.jetbrains.kotlin:kotlin-reflect:$kotlin_version"
+    implementation "com.sparkjava:spark-core:2.7.1"
+    implementation "com.sparkjava:spark-template-freemarker:2.7.1"
+    implementation "org.slf4j:slf4j-simple:1.7.25"
+    implementation "com.google.code.gson:gson:2.2.4"
+    implementation "com.github.salomonbrys.kotson:kotson:2.5.0"
+    implementation "com.opencsv:opencsv:3.9"
+    implementation "org.apache.commons:commons-lang3:3.12.0"
+    implementation "org.apache.commons:commons-text:1.9"
+    implementation "org.postgresql:postgresql:9.4.1212.jre7"
```

Comment:
Postgres seems like an old version, I was wondering if we could use this opportunity to upgrade.

Reply:
Hmm, we shouldn't even need to be using Postgres, this must be superfluous. In general though once we target Java 11 we will need to update a bunch of dependencies.